app_feup/lib/view/Widgets/bug_report_form.dart

```diff
@@ -168,19 +178,47 @@ class BugReportFormState extends State<BugReportForm> {
     );
   }
 
+  Widget consentBox(BuildContext context) {
+    return Container(
+      padding: EdgeInsets.only(),
+      margin: EdgeInsets.only(bottom: 20, top: 0),
+      child: ListTileTheme(
+        contentPadding: EdgeInsets.all(0),
+        child: CheckboxListTile(
+          activeColor: Theme.of(context).primaryColor,
+          title: Text(
+              '''Consinto que a informação seja publicada no GitHub, incluindo o meu contacto pessoal, se fornecido.''',
```

Comment:
Can it be more explicit that it's public? Something like "Consinto que toda esta informação será disponibilizada publicamente na plataforma GitHub, incluindo o meu contacto pessoal, se fornecido". Most importantly, we have to follow the GDPR rules I mentioned above.

Reply:
Sure